website/docs/gator.md

```diff
@@ -240,6 +240,19 @@ the `run` flag:
 gator verify path/to/suites/... --run "disallowed"
 ```
 
+#### Advanced Usage
```

Comment:
I didn't want to call this "advanced usage" but I wanted a blurb that says "careful"

Reply:
Maybe call it "Validating Metadata-Based Constraints" or similar? Not 100% sure what the most user-friendly title is, but it should be as discoverable/recognizable as possible for people who are looking for a solution to "how do I validate a constraint template written against input.review.user" or similar.

We should also mention to people that any constraint templates that require testing against AdmissionReview probably wont work for `gator test` or Gatekeeper's audit functionality, since request metadata does not exist in those contexts.